dots.ipynb

```diff
@@ -0,0 +1,637 @@
+{
```

Comment:
<p>Welcome to the dot calling notebook! </p><p> </p><p>Punctate pairwise peaks of enriched contact frequency are a prevalent feature of mammalian interphase contact maps. These features are also referred to as ‘dots’ or ‘loops’ in the literature, and can appear either in isolation or as parts of grids and at the corners of domains. </p><p> </p><p>HiCCUPS, proposed in <a href="https://doi.org/10.1016/j.cell.2014.11.021" target="_blank">Rao et al. 2014</a>, is a common approach for detecting these features in Hi-C maps. However, HICCUPS is written in Java, making it hard to modify the parameters for specific steps. </p><p> </p><p>Cooltools implements a similar approach for calling dots  in Python. This makes it easy to vary parameters and processing steps for Hi-C or Micro-C datasets.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>